contracts/twap/CachedTwap.sol

```diff
@@ -6,7 +6,7 @@ import { CumulativeTwap } from "./CumulativeTwap.sol";
 
 abstract contract CachedTwap is CumulativeTwap {
     uint256 internal _cachedTwap;
-    uint160 internal _lastUpdatedAt;
+    uint256 internal _lastUpdatedAt;
```

Comment:
wondering why you change the type to `uint256`?  In my understanding, `_lastUpdatedAt` and `_interval` will be loaded in one OP code,  it's more gas-saving.